CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing to Nmstate
+
+:+1: Thank you for contributing! :+1:
+
+The *Nmstate* team is following the guidelines presented in this document.
+These are mostly guidelines, not rules. Use your best judgment and follow
+these guidelines when contributing to the project.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[Nmstate Code of Conduct](CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code.
+Please report unacceptable behavior to the nmstate team.
+
+## How to Contribute
+
+- Bugs: Tracked as ([Jira](https://nmstate.atlassian.net)) issues.
+- Enhancements: RFE suggestions are tracked as
+  ([Jira](https://nmstate.atlassian.net)) issues.
+- Code: Managed on [GitHub](https://github.com/nmstate/nmstate) through
+  Pull Requests ([PR](https://github.com/nmstate/nmstate/pulls)).
+
+#### Pull Requests
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. Run and pass the unit tests and integration tests locally.
+2. Follow the instructions on
+[how to open a PR](https://opensource.guide/how-to-contribute/#opening-a-pull-request).
+3. Follow the [Coding and Style Guidelines](#Coding and Style Guidelines).
+4. After you submit your pull request, verify that all
+[status checks](https://help.github.com/articles/about-status-checks/) are passing.
+
+
+## Coding and Style Guidelines
+
+- Nmstate is written primarily in Python, and its coding style should follow
+  the best practices of Python coding unless otherwise declared.
+- Nmstate uses [black](https://github.com/python/black) code formatter.
+- PEP8 is holy.
+- Tests are holy.
```

Comment:
IMHO this statement is a violation of the proposed CoC, as it states that our standards are `Being respectful of differing viewpoints and experiences` and `Using welcoming and inclusive language`.

Reply:
Same as pep8